zigpy/ota/providers.py

```diff
@@ -540,7 +540,7 @@ async def _load_index(
 
 
 class ThirdReality(BaseOtaProvider):
-    MANUFACTURER_IDS = [4659, 4877]
+    MANUFACTURER_IDS = [4659,5127]
```

Comment:
Was the manufacturer id `4877` not used before?
Also, add the space back.

Reply:
At the request of company leaders, future manufacturer IDs will be 4659 and 5127.

Reply:
Ok, I see that. Just wanted to confirm that no (OTA) devices are currently using `4877`?

Reply:
OK, thanks for the reminder, after communicating again, we decided to keep the current manufacturer IDs and add a new manufacturer ID just in case. I've made the changes in the latest commit.